easy_phi/static/js/main.js

```diff
@@ -21,21 +21,7 @@ var ep = window['ep'] || {
 
     init: function(base_url) {
         // TODO: set global ajax error handler
-        if (base_url == null) {
-            base_url = window.location.protocol + "//" +window.location.host;
-        }
-
-        ep.base_url = base_url;
-
-        setInterval(function(){
```

Comment:
since Platform info is not a side block, we can refresh values only when it is displayed